minidump-processor/src/process_state.rs

```diff
@@ -217,12 +218,18 @@ pub struct ExceptionInfo {
     pub adjusted_address: Option<AdjustedAddress>,
     /// A string representing the crashing instruction (if available)
     pub instruction_str: Option<String>,
+    /// INCOMPLETE: Crashing instruction, later can replace instruction_str
+    pub instruction: Option<Instruction>,
```

Comment:
Putting this here makes this part of [this crate's public API](https://docs.rs/minidump-processor/latest/minidump_processor/struct.ExceptionInfo.html), which isn't great since it's a type from an external crate. It's also an amd64-specific type which doesn't make sense for this struct which is used for minidumps from any architecture.

The `yaxpeax-arch` crate has an [`Instruction` trait](https://docs.rs/yaxpeax-arch/0.3.1/yaxpeax_arch/trait.Instruction.html) that is implemented for this struct (and the parallel structs in other arch-specific yaxpeax crates), but it doesn't provide much functionality.

I'd suggest wrapping this in a newtype or an enum, even if it doesn't look very useful right now, to make it clear that this is an amd64 instruction and avoid directly exposing `yaxpeax` types in this crate's API.

Also, to reiterate the point from my previous comment: this dependency is currently marked as optional, so you need to account for that.